pkg/config/utils.go

```diff
@@ -409,94 +417,151 @@ func checkConfig(atmosConfig schema.AtmosConfiguration, isProcessStack bool) err
 	return nil
 }
 
-func processCommandLineArgs(atmosConfig *schema.AtmosConfiguration, configAndStacksInfo schema.ConfigAndStacksInfo) error {
+const cmdLineArg = "Set using command line argument"
+
+func processCommandLineArgs(atmosConfig *schema.AtmosConfiguration, configAndStacksInfo *schema.ConfigAndStacksInfo) error {
+	if err := setBasePaths(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+	if err := setTerraformConfig(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+	if err := setHelmfileConfig(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+	if err := setStacksConfig(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+	if err := setFeatureFlags(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+	if err := setSchemaDirs(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+	if err := setLoggingConfig(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+	if err := setSettingsConfig(atmosConfig, configAndStacksInfo); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func setBasePaths(atmosConfig *schema.AtmosConfiguration, configAndStacksInfo *schema.ConfigAndStacksInfo) error {
 	if len(configAndStacksInfo.BasePath) > 0 {
 		atmosConfig.BasePath = configAndStacksInfo.BasePath
-		u.LogDebug(fmt.Sprintf("Using command line argument '%s' as base path for stacks and components", configAndStacksInfo.BasePath))
+		log.Debug(cmdLineArg, "BasePath", configAndStacksInfo.BasePath)
 	}
+	return nil
+}
+
+func setTerraformConfig(atmosConfig *schema.AtmosConfiguration, configAndStacksInfo *schema.ConfigAndStacksInfo) error {
 	if len(configAndStacksInfo.TerraformCommand) > 0 {
 		atmosConfig.Components.Terraform.Command = configAndStacksInfo.TerraformCommand
-		u.LogDebug(fmt.Sprintf("Using command line argument '%s' as terraform executable", configAndStacksInfo.TerraformCommand))
+		log.Debug(cmdLineArg, "TerraformCommand", configAndStacksInfo.TerraformCommand)
```

Comment:
I feel like we should convey it not in terms of Atmos go code, but in terms of Atmos configuration (YAML configuration path)

```suggestion
		log.Debug(cmdLineArg, "terraform.command", configAndStacksInfo.TerraformCommand)
```

Reply:
okay got it